compiler/rustc_codegen_gcc/src/context.rs

```diff
@@ -269,10 +266,8 @@ impl<'gcc, 'tcx> CodegenCx<'gcc, 'tcx> {
             struct_types: Default::default(),
             local_gen_sym_counter: Cell::new(0),
             eh_personality: Cell::new(None),
-            rust_try_fn: Cell::new(None),
             pointee_infos: Default::default(),
             structs_as_pointer: Default::default(),
-            cleanup_blocks: Default::default(),
```

Comment:
This is used under the feature `master`, so please keep it (possibly under a `#[cfg(feature="master")]` to not have a warning anymore).